FINS/FINS-IOC-01App/Db/sans2d-vac-status.template

```diff
@@ -50,7 +75,7 @@ record(calcout,"$(FULL_PREFIX)$(SYSTEM):STATUS:VAC_STATUS_CALC" ) {
 
 ### Simulation Records ###
 
-record(mbbi, "$(FULL_PREFIX)SIM:$(SYSTEM):STATUS")
+record(mbbo, "$(FULL_PREFIX)SIM:$(SYSTEM):STATUS")
```

Comment:
This should be an mbbi record.

Reply:
To test this functionality we need to be able to set the statues in recsim somehow. We cannot do this through the actual record as it is an mbbi, so making this an mbbo means we can set it through the SIOL.

Reply:
No, it would work just as well if it was an mbbi. You can set sim PVs even if they are input records.